lib/matplotlib/ticker.py

```diff
@@ -411,6 +411,10 @@ class FormatStrFormatter(Formatter):
 
     The format string should have a single variable format (%) in it.
     It will be applied to the value (not the position) of the tick.
+
+    Negative numeric values will use a dash not a unicode minus,
+    use mathtext to get a unicode minus by wrappping the format specifier
+    with $ (e.g. $%g$).
```

Comment:
```suggestion
    with $ (e.g. "$%g$").
```
Let's mark the example explicitly as a string.

Reply:
Made the suggested change.